src/components/liveSession/services/liveSession.service.js

```diff
@@ -13,12 +13,25 @@
             };
 
             let activeLiveSessionDataFromAdapter = null;
-            let scheduledLessonFromAdapter = null;
             let currUserLiveSessionState = UserLiveSessionStateEnum.NONE.enum;
             let registeredCbToActiveLiveSessionDataChanges = [];
             let registeredCbToCurrUserLiveSessionStateChange = [];
             let liveSessionInterval = {};
             let isTeacherApp = (ENV.appContext.toLowerCase()) === 'dashboard';
+            let scheduledLessonFromAdapter = null;
+
+            this.getScheduledLessonData = (lessonId) => {
```

Comment:
Please change this function's name (getScheduledLessonData) to indicate the returned value is from the saved data in the adapter.